desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java

```diff
@@ -937,6 +906,34 @@ private void activateDisplayPreferences() {
 
         resetDontShowAgainButton.setOnAction(e -> preferences.resetDontShowAgain());
 
+        editCustomBtcExplorer.setOnAction(e -> {
+            EditCustomExplorerWindow urlWindow = new EditCustomExplorerWindow("BTC",
+                    preferences.getBlockChainExplorer(), preferences.getBlockChainExplorers());
+            urlWindow
+                    .actionButtonText("Save")
```

Comment:
I think you forgot here to use a resource string. I guess its: `Res.get("shared.save")`